nixos/services/postgresql/default.nix

```diff
@@ -82,13 +82,14 @@ let
   mkExtensionNamesForAutoUpgrade =
     postgresqlMajor:
     let
-      availableExtensions =
-        lib.mapAttrs' (attrName: package: lib.nameValuePair (lib.getName package) attrName)
-          (
-            builtins.removeAttrs pkgs."postgresql${toString postgresqlMajor}Packages" [
-              "recurseForDerivations"
-            ]
-          );
+      pgSubpackages = fclib.attrsets.filterSuccessfulEval (
```

Comment:
that feels tbh super weird. Is there a reason why a simple
```
pgSubpackages = pkgs.${packages.postgresqlMajor}.pkgs
```

doesn't work? At least the `recurseForDerivations` removal shouldn't be needed anymore